src/main/app/controller/PostController.ts

```diff
@@ -30,7 +31,7 @@ export abstract class PostController<T extends AnyObject> {
 
       await res.locals.storage.store(state);
 
-      res.redirect(this.getNextStep(req.body));
+      res.redirect(getNextStepUrl(req, req.body as Record<string, string>));
```

Comment:
I'm not so sure I'm keen on pulling this in globally? Normally I'd prefer some kind of DI so it can be mocked.

Reply:
👍 Mocked out in `PostController.test.ts`